finetune/adapter.py

```diff
@@ -150,10 +147,12 @@ def train(
         # this assumes that all samples have a fixed length equal to the longest sequence length
         # which is most likely false during finetuning
         x = torch.randint(0, 1, (micro_batch_size, longest_seq_length))
-        measured_flops = measure_flops(meta_model, x)
+        forward_fn = lambda: meta_model(x)
+        measured_flops = measure_flops(meta_model, forward_fn, torch.Tensor.sum)
```

Comment:
Shouldn't you include the loss as well? For example, I did it like this:
https://github.com/Lightning-AI/lit-gpt/pull/677/files#diff-00fedfb359238295e8b95bc7c9f90975463bf4c40f03b2a010a118ec21f2f8d3R163